packages/subgraph/tests/plugin/adminMembers.test.ts

```diff
@@ -23,11 +23,10 @@ const pluginEntityId = generateEntityIdFromAddress(pluginAddress);
 
 describe('AdminMembers', function () {
   // keccack256 of EXECUTE_PROPOSAL_PERMISSION
-  const AdminPermission = EXECUTE_PROPOSAL_PERMISSION_HASH;
 
   beforeEach(function () {
     let context = new DataSourceContext();
-    context.setString('permissionId', AdminPermission);
+    context.setString('permissionId', EXECUTE_PROPOSAL_PERMISSION_HASH);
```

Comment:
```suggestion
    context.setString('permissionId', EXECUTE_PROPOSAL_PERMISSION_ID);
```
@Michael-A-Heuer is this a valid request? for the sake of the naming convention every where?

Reply:
waiting for Michael response